File: TiendaAPI/Models/Ingrediente.cs
using System.ComponentModel.DataAnnotations; 

namespace TiendaAPI.Models
{
    public class Ingrediente
    {
        [Key] 
        public int idIngrediente { get; set; }

        public string? nombre { get; set; }
        public int? stock { get; set; }
        public string? unidadMedida { get; set; }
        public string? tipo { get; set; }
        public decimal? precioUnitario { get; set; }
        public bool? activo { get; set; }
    }
}

File: Frontend/src/interfaces/Ingrediente/Ingrediente.ts
export interface InventarioInterface {
    idInventario: number;
    idIngrediente: number;
    ingrediente: IngredienteInterface;
    cantidad: number;
    tipoMovimiento: string;
    fechaMovimiento: string; 
    detalle: string;
}

export interface IngredienteInterface {
    idIngrediente: number;
    nombre: string;
    stock: number;
    unidadMedida: string;
    tipo: string;
    precioUnitario: number;
    activo: boolean;
}

